docs/modding/suitmodding/addinglocals.mdx

```diff
@@ -0,0 +1,666 @@
+- Season 1 suits: `00985A1100E5EDDC`,
+- Season 2 suits: `009F430D046716BE` ,
+- Season 3 suits: `004B8C5124A49543` 
+
+
+<details>
+<summary>about **HASH**</summary>
+:::info HASH
+You can interpret <Tooltip text={<strong>HASH</strong>} tooltip="A unique identifier that acts as the address or coordinates for a file." /> lines as a file's specific address or coordinates. They are the file's *unique identifiers* and can bring us straight to the file.
+>You will constantly have to use these throughout modding.
+:::
+</details>
+[[Return to Adding More LOCR files]](#adding-more-locr-files)
+### 2. Go to your LOCR  
+
+- copy  your relevant <Tooltip text={<strong>HASH</strong>} tooltip="A unique identifier that acts as the address or coordinates for a file." />  
+- Go to the <Tooltip text={<strong>game contents tab</strong>} tooltip="A tab in Glacierkit where you search game files" />
+- paste it on the search bar and hit enter.
+>You'll see a map tree or *file path* leading to a <Tooltip text={<><strong>LOCR File</strong><span className="icon-alert"></span></>} tooltip="A file that holds game text in different languages, linked by unique 'strings'." />.
+- <Tooltip text={<italic>Open it</italic>} tooltip="By clicking it" />
+
+:::tip LOCR FILE
+
+A <Tooltip text={<strong>LOCR File</strong>} tooltip="A file that holds game text in different languages, linked by unique 'strings'." /> can be identified with 2 attributes:
+    - named like `randomwords.sweetmenutext].pc_localized-textlist` 
+    - having a language or translation icon.
+:::
+
+
+
+### 3. Confirm your LOCR
+*You will see a *Preview* text box with code on the right and* <Tooltip text={<strong>additional info on top</strong>} tooltip="The file's properties are displayed on top after clicking it" />.
+
+Confirm you have the right file by checking:
+- `LOCR` type of file 
+- `HASH` code.
+
+<details>
+<summary>***Advanced Search***</summary>
+:::info advanced SEARCH
+To speed up the process of finding the names and descriptions of suits, I have given the relevant file's <Tooltip text={<strong>HASHES</strong>} tooltip="A unique identifier that acts as the address or coordinates for a file." /> for now.  
+>To locate any other text in-game, use the [**Advanced Search Guide**](#advanced-search-guide)  
+
+*After following that you will have your* <Tooltip text={<strong>LOCR File</strong>} tooltip="A file that holds game text in different languages, linked by unique 'strings'." /> *and a link that leads to the next part of tutorial.*  
+:::
+</details>
+
+---
+
+## Pt. 2: **GET YOUR STRING**
+*Every line of code in a* <Tooltip text={<strong>LOCR File</strong>} tooltip="A file that holds game text in different languages, linked by unique 'strings'." /> *consists of a* 
+>   - <Tooltip text={<><strong>string</strong><span className="icon-alert"></span></>} tooltip= "A key that links to text in the LOCR File" /> 
```

Comment:
Technically a localisation string is the actual text in English/French/whatever? The distinction is not super important but iirc the game calls the `UI_TEXT_XYZ` "localisation keys"

Reply:
I'll use that term then instead of strings. But a distinction had to be made between the two hash types of the file and the "strings" inside